renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "schedule": [
+    "before 5am on Monday"
```

```suggestion
    "after 2am and before 3am"
```

Why not every day? ;)

---

It causes too much of a mess, especially in monorepo and especially when some of the packages get released multiple times a day. Once a week update cycle has been tested by me in the previous company and by SFUI team for last few months and works like a charm 😄

---

Ok, make sense 👍